src/test/java/com/cloudbees/jenkins/support/filter/InetAddressContentFilterTest.java

```diff
@@ -76,6 +78,7 @@ public void shouldFilterInetAddresses() {
     @Test
     public void shouldNotFilterInetAddressMatchingJenkinsVersion() {
         Jenkins.VERSION = "1.2.3.4";
+        resetMappings();
```

Comment:
Why do you need this if there is a before method?

Reply:
Here we're changing the Jenkins version. Reset is what causes the stop words to be picked and include the Jenkins version we specified on the line above.